themes/default/content/blog/pulumi-security-posture/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Pulumi Security Posture"
```

Comment:
Suggest a title like "Using Pulumi Securely" since "Pulumi Security Posture" could lead the user to believe that the article is about our security posture as a company.

Reply:
Updated the title